crates/typed-store/src/rocks/mod.rs

```diff
@@ -1862,6 +1863,47 @@ pub fn point_lookup_db_options() -> DBOptions {
     db_options
 }
 
+/// Use it only for tables which observe high write rate and grow quickly in size
+pub fn optimized_for_high_throughput_options(
+    block_cache_size_mb: usize,
+    optimize_for_point_lookup: bool,
+) -> DBOptions {
+    let mut db_options = default_db_options();
+    db_options.options.set_write_buffer_size(128 * 1024 * 1024);
+    db_options.options.set_min_write_buffer_number_to_merge(2);
+    db_options.options.set_max_write_buffer_number(6);
+    db_options
+        .options
+        .set_level_zero_file_num_compaction_trigger(2);
```

Comment:
This reduces the default from 4?

Reply:
Yeah, we don't want to wait too long (i.e. for 4 files) for tables with high write rate to start compaction to level 1. So reducing it to 2